composer.json

```
@@ -1,66 +1,156 @@
 {
 	"name": "pfefferle/wordpress-activitypub",
-	"description": "The ActivityPub protocol is a decentralized social networking protocol based upon the ActivityStreams 2.0 data format.",
+	"description": "A WordPress implementation of the ActivityPub protocol, enabling decentralized social networking using the ActivityStreams 2.0 data format.",
 	"type": "wordpress-plugin",
+	"keywords": [
+		"wordpress",
+		"activitypub",
+		"fediverse",
+		"decentralized",
+		"social-network"
+	],
+	"homepage": "https://github.com/pfefferle/wordpress-activitypub",
+	"license": "MIT",
+	"authors": [
+		{
+			"name": "Matthias Pfefferle",
+			"email": "pfefferle@gmail.com",
+			"homepage": "https://notiz.blog",
+			"role": "Lead Developer"
+		}
```

If we add authors, we should add Konstantin and Automattic!

```suggestion
		{
			"name": "Automattic",
			"homepage": "https://automattic.com",
			"role": "Developer"
		},
		{
			"name": "Matthias Pfefferle",
			"homepage": "https://notiz.blog",
			"role": "Developer"
		},
		{
			"name": "Konstantin Obenland",
			"homepage": "https://obenland.it",
			"role": "Developer"
		}
```